app/assets/stylesheets/govuk_publishing_components/components/_notice.scss

```diff
@@ -4,7 +4,7 @@
   @include govuk-responsive-margin(8, "bottom");
 
   clear: both;
-  border: 2px solid govuk-colour("blue");
+  border: 5px solid govuk-colour("blue");
```

Comment:
Can we use `$govuk-border-width`? not sure how the other ones are being set, but happy to update them as a follow up.

Reply:
Nice, I didn't know about that! Will amend

Reply:
nice one! yeah hidden docs here https://govuk-frontend-review.herokuapp.com/docs/#settings/measurements-variable-govuk-border-width